src/plugins/discover/public/application/main/components/top_nav/discover_topnav.tsx

```diff
@@ -201,8 +201,13 @@ export const DiscoverTopNav = ({
 
   const searchBarCustomization = useDiscoverCustomization('search_bar');
 
+  const SearchBar = useMemo(
```

Comment:
Since we can't easily cover this with a functional test in `discover_customization_examples`, can you add a simple Jest test to ensure that `searchBarCustomization.CustomSearchBar` is rendered instead of `AggregateQueryTopNavMenu` when a custom implementation is provided?

Reply:
Done.